common/inc/stdc++/linux/exception

```diff
@@ -33,11 +33,11 @@ namespace std
 	class exception
 	{
 	public:
-		exception() throw();
```

Comment:
I don't get any failures without  the changes in exception and typeinfo. Are the changes  necessary?  If necessary, exception.cc also need to be changed accordingly.